var/spack/repos/builtin/packages/py-keras/package.py

```diff
@@ -39,8 +40,12 @@ class PyKeras(PythonPackage):
     version('1.1.1', sha256='be1b67f62e5119f6f24a239a865dc47e6d9aa93b97b506ba34cab7353dbc23b6')
     version('1.1.0', sha256='36d83b027ba9d2c9da8e1eefc28f600ca93dc03423e033b633cbac9061af8a5d')
 
-    depends_on('py-keras-applications', type='run', when='@2.2:')
-    depends_on('py-keras-preprocessing', type='run', when='@2.2:')
+    depends_on('python@3.6:', type=('build', 'run'), when='@2.4:')
+    depends_on('py-numpy@1.9.1:', type=('build', 'run'), when='@2.4:')
+    depends_on('py-scipy@0.14:', type=('build', 'run'), when='@2.4:')
+    depends_on('py-h5py', type=('build', 'run'), when='@2.4:')
+    depends_on('py-keras-applications', type='run', when='@2.2')
+    depends_on('py-keras-preprocessing', type='run', when='@2.2')
     depends_on('py-setuptools', type='build')
     depends_on('py-theano', type=('build', 'run'))
```

Comment:
theano and six are no longer required in the latest version

Reply:
Where did you get the install dependencies? I was searching online but couldn't find them.

Reply:
https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#python-dependencies

Reply:
Thanks.